ext/gmp/tests/gh16870.phpt

```diff
@@ -0,0 +1,24 @@
+--TEST--
+Bug GH-16870 (Overly restrictive gmp_pow overflow checks)
+--EXTENSIONS--
+gmp
+--FILE--
+<?php
```

Comment:
```suggestion
--EXTENSIONS--
gmp
--XFAIL--
Currently fails after the introduction of overflow warning
--FILE--
<?php
```

Reply:
Hi,

CI is failing because gmp in `master` is broken.  Can't compute 2^64.

```bash
$ ./sapi/cli/php -a
Interactive shell

php > print gmp_strval(gmp_pow(2, 63));
9223372036854775808
php > print gmp_strval(gmp_pow(2, 64));

Warning: Uncaught ValueError: base and exponent overflow in php shell code:1
Stack trace:
#0 php shell code(1): gmp_pow(2, 64)
#1 {main}
  thrown in php shell code on line 1
php > 
```

Reply:
Yes I know, but this probably needs to backported to 8.2 (or the lowest branch where this occurs) and adding as XFAILs allows the CI to pass so that one can write the proper bug fix.

No PR is going to get merged with a red CI.

Reply:
These tests shouldn't be failing.

Reply:
You are missing the point.
An XFAIL is here to indicate that these test *should* pass but are currently failing. However, having CI being red means that other PRs cannot rely on CI.